### DISCUSSION TITLE
add Interactive Graph Cuts [Feature]

### DISCUSSION THREAD

**Discussion by fanweiya** [wkentaro/labelme#1104] [Ideas / Feature requests]
add Interactive Graph Cuts or Grab Cut

**Comment by Luk3M** [comment]
I think this would be useful for quick segmentation.

I think [this OpenCV tutorial](https://docs.opencv.org/master/d8/d83/tutorial_py_grabcut.html) could be a good reference.

**Comment by kvnptl** [comment]
Is this integrated into the tool or not yet?
